src/components/Alphabet/Letter/Letter.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import styles from './Letter.css';
+
+const checkLetter = (letter, id, clickedLetter) => {
+    const url = `http://localhost:3000/phrases/check?id=${id}&letter=${letter}`
+    return fetch(url, {
+        method: 'post',
+    })
+    .then(response => response.json())
+    .catch(error => {
+        console.error(error);
+    })
+    .then(data => {
+        clickedLetter(data);
```

Comment:
This is a bad name for a function - clickedLetter would suggest that it contains 'the letter that was clicked' not a function!

Reply:
What do you suggest? checkClickedLetter, handleClickedLetter?

Reply:
onLetterClick for the prop name, handleLetterClicked for method name